components/Markdown/Markdown.css

```diff
@@ -1,55 +1,259 @@
-.msrd .msrd-Markdown h1 {
-  font: var(--font-12);
-  font-weight: 800;
-  letter-spacing: var(--font-12-letter-spacing);
-  margin: 0;
-  margin-bottom: var(--space-vertical-04);
+.msrd .msrd-Markdown {
+  font: var(--font-03);
+  letter-spacing: var(--font-03-letter-spacing);
 }
 
-.msrd .msrd-Markdown h2 {
-  font: var(--font-10);
-  font-weight: 800;
-  letter-spacing: var(--font-10-letter-spacing);
-  margin: 0;
-  margin-bottom: var(--space-vertical-04);
+.msrd .msrd-Markdown--measured {
+  max-width: 34em;
 }
 
-.msrd .msrd-Markdown h3 {
-  font: var(--font-08);
-  font-weight: 800;
-  letter-spacing: var(--font-08-letter-spacing);
-  margin: 0;
-  margin-bottom: var(--space-vertical-04);
+.msrd .msrd-Markdown--inline {
+  font: inherit;
+  letter-spacing: inherit;
+  max-width: 100%;
 }
 
-.msrd .msrd-Markdown h4 {
-  font: var(--font-06);
+@media (min-width: 48em) {
+  .msrd .msrd-Markdown {
+    font: var(--font-04);
+    letter-spacing: var(--font-04-letter-spacing);
+  }
+
+  .msrd .msrd-Markdown--inline {
+    font: inherit;
+    letter-spacing: inherit;
+  }
+}
+
+.msrd .msrd-Markdown blockquote {
+  margin-bottom: var(--space-vertical-03);
+  margin-left: 1em;
+  margin-right: 0;
+  margin-top: 0;
+}
+
+.msrd .msrd-Markdown blockquote p {
+  margin-bottom: var(--space-vertical-03);
+  quotes: "“" "”" "‘" "’";
+}
+
+.msrd .msrd-Markdown blockquote p:first-child::before {
+  content: open-quote;
+  margin-left: -0.66ch;
+}
+
+.msrd .msrd-Markdown blockquote p:first-child::after {
+  content: close-quote;
+}
+
+.msrd .msrd-Markdown blockquote p + p:last-child {
+  text-align: right;
+  font: var(--font-02);
+}
+
+@media (min-width: 48em) {
+  .msrd .msrd-Markdown blockquote p + p:last-child {
+    font: var(--font-03);
+  }
+}
+
+.msrd .msrd-Markdown code {
```

Comment:
I would probably expect this to scroll on overflow-x, rather than wrap the text

<img width="774" alt="image" src="https://github.com/measuredco/measuredco-site/assets/985961/b4332bc3-7dcc-41c0-9bc7-24fb5fe18559">

Reply:
Done but scroll needs a little more finessing. Planning to cover this when adding syntax highlighting.